src/command_modules/azure-cli-rdbms/tests/test_rdbms_commands.py

```diff
@@ -0,0 +1,354 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from datetime import datetime
+from dateutil.tz import tzutc
+
+from azure.cli.testsdk.base import execute
+from azure.cli.testsdk.exceptions import CliTestError
+from azure.cli.testsdk import (
+    JMESPathCheck,
+    NoneCheck,
+    ResourceGroupPreparer,
+    ScenarioTest)
+from azure.cli.testsdk.preparers import (
+    AbstractPreparer,
+    SingleValueReplacer)
+
+from azure.cli.testsdk.utilities import create_random_name
+
+
+# Constants
+server_name_prefix = 'clitestserver'
+server_name_max_length = 63
+
+
+class ServerPreparer(AbstractPreparer, SingleValueReplacer):
+    # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-instance-attributes
+    def __init__(self, engine_type='mysql', engine_parameter_name='engine',
```

Comment:
`engine` is a generic name, it is recommended to be a more specific name. for example, `database_engine`